Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -402,6 +402,39 @@ public HttpResponseMessage PatchAgreementConcludedAt(int id, [FromBody] SingleVa
                 .Match(_ => Ok(), FromOperationError);
         }
 
+        [HttpPatch]
+        [Route("{id}/oversight-option")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage PatchOversightOption(int id, [FromBody] SingleValueDTO<YearMonthIntervalOption> oversightInterval)
+        {
+            if (oversightInterval == null)
+                return BadRequest("YearMonthIntervalOption must provided");
+            
+
+            return _dataProcessingRegistrationApplicationService
+                .UpdateOversightInterval(id, oversightInterval.Value)
+                .Match(_ => Ok(), FromOperationError);
+        }
+
+        [HttpPatch]
+        [Route("{id}/oversight-option-note")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage PatchOversightOptionNote(int id, [FromBody] SingleValueDTO<string> oversightIntervalNote)
+        {
+            if (oversightIntervalNote == null)
+                return BadRequest("Note must be provided");
```

Comment:
det hedder parameteren ikke - brug nameof of string interpolation. Så følges evt. navngivninger ad